<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mute Exceptions</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="mainLayout">
      <item>
       <layout class="QVBoxLayout" name="mutedLayout">
        <item>
         <widget class="QLabel" name="mutedLabel">
          <property name="text">
           <string>Muted</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="mutedListWidget"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QFormLayout" name="buttonsLayout">
        <property name="formAlignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <item row="0" column="0">
         <widget class="QPushButton" name="addExceptionButton">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>Add Exception</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="removeExceptionButton">
          <property name="text">
           <string>Remove Exception</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="exceptionsLayout">
        <item>
         <widget class="QLabel" name="exceptionsLabel">
          <property name="text">
           <string>Exceptions</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="exceptionsListWidget"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="applyButton">
      <property name="text">
       <string>Apply</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
